spec/marten/http/status_spec.cr

```diff
@@ -0,0 +1,22 @@
+require "./spec_helper"
+
+describe Marten::HTTP::Status do
+  describe ".status_code" do
+    it "accepts integer status code" do
+      status = Marten::HTTP::Status.status_code(400)
+      status.should eq 400
+    end
+
+    it "accepts symbol status code" do
+      status = Marten::HTTP::Status.status_code(:internal_server_error)
+      status.should eq 500
```

Comment:
Would it be possible to iterate over all the <symbol, code> pairs and verify that they produce the expected result?

For example:

```crystal
  [
    [:internal_server_error, 500],
    [:forbidden, 403],
    # ...
  ].each do |code_symbol, expected_status_code|
    Marten::HTTP::Status.status_code(code_symbol).should eq expected_status_code
  end
```